src/test/kotlin/no/nav/syfo/aktivitetskrav/cronjob/OutdatedAktivitetskravCronjobSpek.kt

```diff
@@ -70,7 +70,7 @@ class OutdatedAktivitetskravCronjobSpek : Spek({
         }
         it("Lukker nytt aktivitetskrav hvor stoppunkt er etter arena-cutoff og før outdated-cutoff") {
             val aktivitetskrav = createNyttAktivitetskrav(
-                stoppunktAt = arenaCutoff.plusDays(1)
+                stoppunktAt = arenaCutoff.plusDays(2)
```

Comment:
Testen sier jo bare "etter arena-cutoff" og det stemmer jo fortsatt. Men det er ikke åpenbart at det nå er pluss 2 dager fordi vi et annet sted i koden trekker fra 1 dag. Er det dumt, eller er testen grei likevel?

Reply:
Kanskje bedre å endre createNyttAktivitetskrav-metoden slik at tilfellestart blir satt riktig ut i fra stoppunkt-datoen som sendes inn her?

Reply:
Ja, jeg prøver det i stedet 👍

Reply:
Det er for øvrig flere tester her jeg ikke måtte endre, selv om de også setter stoppunktAt til arenaCutoff.plusDays(1) 🤷